extensions/wikia/AdEngine/tests/AdEngine2ContextServiceTest.php

```diff
@@ -32,16 +32,6 @@ private function getTitleMock( $type, $langCode, $artId, $artDbKey ) {
 		return $title;
 	}
 
-	private function mockMonetizationModule() {
-		$name = 'MonetizationModuleHelper';
-		$mock = $this->getMock( $name );
-		$mock->method( 'getMonetizationUnits' )->willReturn( ['below_category' => 'testing'] );
-		$mock->method( 'getWikiVertical' )->willReturn( 'other' );
-		$mock->method( 'getCacheVersion' )->willReturn( 'v1' );
-		$this->mockClass( $name, $mock );
-		$this->mockStaticMethod( $name, 'canShowModule', true );
-	}
-
 	public function adContextDataProvider() {
 		$defaultParameters = [
 			'titleMockType' => 'article',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "titleMockType" 17 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "article" 18 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)